src/front/app/components/Challenges/Challenge.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Challenge = ({ description, user, location }) => (
+    <li className="challenge">
+        <div className="challenge__image">Image here</div>
+        <span className="challenge__description">
+            {description && description.toUpperCase()}
```

Comment:
Consider setting required props in propTypes (or default values for optional props) instead of such repetitive conditioning